_episodes/03-monitor-the-model.md

```diff
@@ -356,27 +358,31 @@ For the present *regression task*, it makes more sense to compare true and predi
 So, let's look at how the predicted sunshine hour have developed with reference to their ground truth values.
 
 ~~~
-fig, axes = plt.subplots(1, 2, figsize=(12, 6))
-plt.style.use('ggplot')  # optional, that's only to define a visual style
-axes[0].scatter(y_train_predicted, y_train, s=10, alpha=0.5, color="teal")
-axes[0].set_title("training set")
-axes[0].set_xlabel("predicted sunshine hours")
-axes[0].set_ylabel("true sunshine hours")
+# We define a function that we will reuse in this lesson
+def plot_predictions(y_pred, y_true, title):
```

Comment:
It's more intuitive to use actual values as x, predicted as y. In this way one could easily see if the predicted is larger than actual value by checking if the position of predicted value is higher than diagonal or not. "Higher is larger" is more intuitive than "more right is larger" :-)

Reply:
Sounds good to me

Reply:
I think both are used in the field. But I get your reasoning. I created an issue for it: #289